src/Altinn.Platform/Altinn.Platform.Events/Tests/TestingControllers/EventsSubscriptionControllerTests .cs

```diff
@@ -0,0 +1,248 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using Altinn.Common.AccessToken.Services;
+using Altinn.Common.PEP.Interfaces;
+using Altinn.Platform.Events.Controllers;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Repository.Interfaces;
+using Altinn.Platform.Events.Services.Interfaces;
+using Altinn.Platform.Events.Tests.Mocks;
+using Altinn.Platform.Events.Tests.Mocks.Authentication;
+using Altinn.Platform.Events.Tests.Utils;
+using Altinn.Platform.Events.UnitTest.Mocks;
+using AltinnCore.Authentication.JwtCookie;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Moq;
+using Xunit;
+
+namespace Altinn.Platform.Events.Tests.TestingControllers
+{
+    /// <summary>
+    /// Represents a collection of integration tests.
+    /// </summary>
+    public partial class IntegrationTests
+    {
+        /// <summary>
+        /// Represents a collection of integration tests of the <see cref="EventsController"/>.
+        /// </summary>
+        public class EventsSubscriptionControllerTests : IClassFixture<WebApplicationFactory<Startup>>
```

Comment:
EventsSubscription here as well.. And some mis match in the naming and comments.. Ref eventsControllerTests in one of  the comments.

Reply:
Fixed